sample_size/utils/sample_size_run.py

```diff
@@ -0,0 +1,43 @@
+from sample_size.sample_size_calculator.sample_size_calculator import SampleSizeCalculators
```

Comment:
since this implementation is independent of the sample size calculator, how about putting it outside, putting it under utils directory seems like it has some functions that is needed by `sample_size_calculator`
we can add a scripts directory, something like
```
sample-size-repo/scripts/run/sample_size_run.py
                             utils.py
                             __init__.py
```